src/platform/datapath_winuser.c

```diff
@@ -3980,7 +3560,6 @@ CxPlatDataPathSocketProcessRioCompletion(
 {
     UNREFERENCED_PARAMETER(Cqe);
     CXPLAT_SOCKET_PROC* SocketProc = CONTAINING_RECORD(Sqe, CXPLAT_SOCKET_PROC, RioSqe);
-    CXPLAT_DATAPATH* Datapath = SocketProc->DatapathProc->Datapath;
```

Comment:
Why?

Reply:
Maybe the "Datapath" is used only once, but let me  change back